src/features/override-emotes.js

```diff
@@ -32,7 +32,7 @@ module.exports = function() {
                         } else if ($emote.data('channel') && $emote.data('channel') === 'BetterTTV Emotes') {
                             return 'Emote: ' + raw + '<br />BetterTTV Emoticon';
                         } else if ($emote.data('channel')) {
-                            return 'Emote: ' + raw + '<br />Channel: ' + $emote.data('channel');
+                            return 'Emote: ' + raw + '<br />Channel: ' + $emote.data('channel') + ' (BTTV)';
```

Comment:
every single channel-specific twitch emotes will have their `data-id` set to some emote in `TwitchEmoteIDToChannel`?